web/angular-wallet/src/app/login/login.component.ts

```diff
@@ -1,4 +1,4 @@
-import 'particles.js';
+import '../../particles';
```

Comment:
needed to incorporated the source, as the package is not very active and the PR wasn't merged yet

Reply:
what if we forked it, applied our change there, and overrode the location to it in `package.json` to point to our fork?

I try to avoid checking in vendor files. This way, if it ever gets merged in upstream, we can just update package.json and be good.

Reply:
I agree with you, but the particle.js project seems to be inactive. 39 PRs open, and no response from the maintainer...